tests/acceptance/onboarding-test.js

```diff
@@ -38,99 +40,99 @@ test('A user can onboard as expected', (assert) => {
 
   authenticateSession(application, { user_id: user.id });
 
-  visit('/');
+  indexPage.visit();
 
   andThen(() => {
     assert.equal(currentURL(), '/start/hello');
-    assert.equal(find('.start-actions button').is(':disabled'), true);
-    fillIn('input[name="firstName"]', 'Josh');
-    fillIn('input[name="lastName"]', 'Smith');
+    assert.ok(onboardingPage.startButton.isDisabled, 'start button disabled by default');
+    onboardingPage.firstName('Josh');
+    onboardingPage.lastName('Smith');
   });
 
   andThen(() => {
-    assert.equal(find('.start-actions button').is(':disabled'), false);
-    click('.start-actions button');
+    assert.notOk(onboardingPage.startButton.isDisabled, 'start button is enabled');
+    onboardingPage.startButton.click();
   });
 
   andThen(() => {
     assert.equal(currentURL(), '/start/interests');
-    assert.equal(find('.start-actions button').is(':disabled'), true);
-    click('.category-item button');
+    assert.ok(onboardingPage.startButton.isDisabled, 'start button is disabled');
+    onboardingPage.clickCategoryItem();
   });
 
   andThen(() => {
-    assert.equal(find('.start-actions button').is(':disabled'), false);
-    click('.category-item button');
+    assert.notOk(onboardingPage.startButton.isDisabled, 'start button is enabled');
+    onboardingPage.clickCategoryItem();
   });
 
   andThen(() => {
-    assert.equal(find('.start-actions button').is(':disabled'), true);
-    click('.category-item button');
+    assert.ok(onboardingPage.startButton.isDisabled, 'start button is disabled');
+    onboardingPage.clickCategoryItem();
   });
 
   andThen(() => {
-    click('.start-actions button');
+    onboardingPage.startButton.click();
```

Comment:
Should there be another assertion before this `click` that `startButton` is clickable?